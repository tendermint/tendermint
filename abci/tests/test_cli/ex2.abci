check_tx 0x00
check_tx 0xff
deliver_tx 0x00
check_tx 0x00
deliver_tx 0x01
deliver_tx 0x04
info
