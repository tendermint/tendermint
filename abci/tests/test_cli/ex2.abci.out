> check_tx 0x00
-> code: OK

> check_tx 0xff
-> code: OK

> finalize_block 0x00
-> code: OK
-> code: OK
-> data.hex: 0x0200000000000000

> commit 
-> code: OK

> check_tx 0x00
-> code: OK

> finalize_block 0x01
-> code: OK
-> code: OK
-> data.hex: 0x0400000000000000

> commit 
-> code: OK

> finalize_block 0x04
-> code: OK
-> code: OK
-> data.hex: 0x0600000000000000

> commit 
-> code: OK

> info 
-> code: OK
-> data: {"size":3}
-> data.hex: 0x7B2273697A65223A337D

